src/ui/pages/NotificationDetails/components/ReceiveCredential/ReceiveCredential.tsx

```diff
@@ -180,7 +182,15 @@ const ReceiveCredential = ({
     try {
       const startTime = Date.now();
       setInitiateAnimation(true);
-      await Agent.agent.ipexCommunications.acceptAcdc(notificationDetails.id);
+
+      // @TODO - foconnor: Should be refined in the upcoming UI ticket
+      //   If multisigMemberStatus.members.length && multisigMemberStatus.members[0] === identifier?.multisigManageAid, we can call admitAcdc
+      //   Otherwise, can call joinMultisigAdmit IF multisigMemberStatus.linkedGroupRequest.current !== undefined
+      if (multisigMemberStatus.linkedGroupRequest.current) {
+        await Agent.agent.ipexCommunications.joinMultisigAdmit(notificationDetails.id);
+      } else {
+        await Agent.agent.ipexCommunications.admitAcdc(notificationDetails.id);
+      }
```

Comment:
@Sotatek-DukeVu @sdisalvo-crd For now this works so that any member can accept first, and the others join. So we need to change this in the next UI PR to only allow the initiator to call admit first, and then the others to join after. Just wanted to do the minimal UI changes to make everything work.

Reply:
LGTM.